portfolio/src/main/webapp/index.html

```diff
@@ -21,13 +21,17 @@ <h1>Sara's Portfolio</h1>
       <div>
         <p>Click <a href="/data">here</a> to view content generated by a servlet.</p>
       </div>
-      <div onload="getComments"></div>
-      <form action="/data" method="POST">
-        <textarea name="comment" rows="10" cols="30">Enter your comment here.</textarea>
-        <br><br>
-        <input type="submit">
-      </form>
-      <div id="comment-data"></div>
+      <textarea id="text" rows="10" cols="30">Enter your comment here.</textarea>
+      <br><br>
+        <select id="language">
+            <option value="es">English</option>
+            <option value="zh">Chinese</option>
+            <option value="es">Spanish</option>
+            <option value="hi">Hindi</option>
+             <option value="ar">Arabic</option>
```

Comment:
nit: remove 1 space to the left, so the <option> is aligned.

Reply:
done